node/src/cli.rs

```diff
@@ -53,18 +53,18 @@ pub enum Cli {
         #[clap(flatten)]
         storage_options: storage::Options,
         /// At minimum, how many triples to stockpile on this node.
-        #[arg(long, env("MPC_RECOVERY_MAX_TRIPLES"), default_value("2"))]
+        #[arg(long, env("MPC_RECOVERY_MIN_TRIPLES"), default_value("20"))]
         min_triples: usize,
         /// At maximum, how many triples to stockpile on this node.
-        #[arg(long, env("MPC_RECOVERY_MAX_TRIPLES"), default_value("10"))]
+        #[arg(long, env("MPC_RECOVERY_MAX_TRIPLES"), default_value("120"))]
```

Comment:
should be at least a factor of 10 for the max. So something like 400 would be better if this is our prod value going forward